src/main/java/CarRacing.java

```diff
@@ -0,0 +1,9 @@
+public class CarRacing {
+    public static void executeGame(InputView input) {
+        final Cars cars = Cars.createCars(input.carCount);
```

Comment:
InputView를 굳이 인자로 전달하지 말고 다음과 같이 구현해도 되지 않을까요?


```suggestion
        final int carCount = InputView.carCount();
        final int gameCount = InputView.gameCount();
```